util/report_utils/compare_results.rb

```diff
@@ -19,16 +19,16 @@ def write_csv(output_path, result_a, result_b)
 
   # TODO: use release names
   CSV.open(output_path.to_s, "wb") do |csv|
-    csv << ["", "$RELEASE_A_NAME ($RELEASE_A_NUMBER)", "", "",
-            "$RELEASE_B_NAME ($RELEASE_B_NUMBER)", "", ""]
-    csv << ["Duration", "max ms", "mean ms", "std dev", "max ms", "mean ms",
+    csv << ["", "$RELEASE_A_NAME ($RELEASE_A_NUMBER)", "", "", "",
+            "$RELEASE_B_NAME ($RELEASE_B_NUMBER)", "", "", "", ""]
+    csv << ["Duration (ms)", "min", "max", "mean", "std dev", "min", "max", "mean",
             "std dev", "% (mean) diff"]
 
     row_labels.each_with_index do |item, index|
       res_index = index + 1
-      csv << [item, result_a[res_index][1], result_a[res_index][2], result_a[res_index][3],
-              result_b[res_index][1], result_b[res_index][2], result_b[res_index][3],
-              percent_diff_string(result_a[res_index][2], result_b[res_index][2])]
+      csv << [item, result_a[res_index][1], result_a[res_index][2], result_a[res_index][3], result_a[res_index][4],
```

Comment:
That is a lot of hard-coded indexes.  It there a more resilient way to write this?
e.g.
```
data_start_index = 1
data_end_index = 4
magic_index = 3

percent_diff = percent_diff_string(result_a[res_index][magic_index], result_a[res_index][magic_index])

csv << [item,
        result_a[data_start_index..data_end_index],
        result_b[res_index][data_start_index..data_end_index],
        percent_diff].flatten
```

Reply:
Thanks; I've update them to use variable names based on the column names.